lib/src/value/calculation.dart

```diff
@@ -604,6 +604,21 @@ final class SassCalculation extends Value {
     }
   }
 
+  /// Creates an `calc-size()` calculation with the given [basis] and [value].
+  ///
+  /// The [basis] and [value] must be either a [SassNumber], a
+  /// [SassCalculation], an unquoted [SassString], or a [CalculationOperation].
+  ///
+  /// This automatically simplifies the calculation. It throws an exception if
+  /// it can determine that the calculation will definitely produce invalid CSS.
+  static SassCalculation calcSize(Object basis, Object? value) {
+    var args = [basis, if (value != null) value];
```

Comment:
Should you check for types here? That they are either a `SassNumber`, a `SassCalculation`,  `SassString`, or a `CalculationOperation`?

Reply:
That's checked by `_simplify()`.